docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -250,6 +669,23 @@ The following table lists all the server metadata attributes the client can rece
 | `MC_META_MEDIA_TRACK_NUM`   | Track number of the latest content in the media controller server |
 | `MC_META_MEDIA_PICTURE`     | Album art of the latest content in the media controller server |
 
+<a name="playlist_updatemode"></a>
+## Media Controller Playlist Update Mode Attributes
+
+The following table lists all the playlist update mode attributes the client can receive.
+
+**Table: Media controller playlist update mode attributes**
+
+| Attribute                        | Description                              |
+|----------------------------------|------------------------------------------|
+| **Playlist update modes**        |                                          |
+| `MC_PLAYLIST_UPDATED`            | Create or updated playlist               |
+| `MC_PLAYBACK_REMOVED`            | Remove playlist                          |
+
+**Note**
```

Comment:
From line 685 to line 687, please add "> " in front of the lines.
```
> **Note**
>
> This Attributes support Tizen 4.0 and higher for Mobile.
```

Reply:
I added "> " in front of the Note.